ddl/rollingback.go

```diff
@@ -414,8 +415,8 @@ func convertJob2RollbackJob(w *worker, d *ddlCtx, t *meta.Meta, job *model.Job)
 			job.Error = toTError(err)
 		}
 		if !job.Error.Equal(errCancelledDDLJob) {
-			job.Error = job.Error.Class().Synthesize(job.Error.Code(),
-				fmt.Sprintf("DDL job rollback, error msg: %s", job.Error.ToSQLError().Message))
+			job.Error = terror.GetErrClass(job.Error).Synthesize(terror.ErrCode(job.Error.Code()),
```

Comment:
Can we use `errors.Annotatef` here directly? @wjhuang2016

Reply:
Here `job.Error` should be a `terror.Errors` type, but `errors.Annotatef` will produce an `error` type.